readme.adoc

```diff
@@ -883,7 +888,7 @@ Above does require using a shell that allows for variable evaluation, if you are
 jbang edit --open=[editor] helloworld.java
 ----
 
-The editor used will be what is specified as the argument to `--open` or default to `$JBANG_EDITOR`, `$VISUAL` or `$EDITOR` in that order.
```

Comment:
Btw, I have always doubted was would happen exactly here: does setting JBANG_EDITOR mean that automatically `--open` is assumed and an editor will be opened or do you still need to provide `--open`? (The latter makes more sense to me but the wording seems to suggest the former)

Reply:
--open needed.